client/src/modules/inventory/list/modals/search.modal.html

```diff
@@ -10,106 +7,109 @@
     </ol>
   </div>
 
-  <div class="modal-body">
-
-    <fieldset>
-      <div class="form-group">
-        <label class="control-label" translate>
-          FORM.LABELS.GROUP
-        </label>
-        <bh-clear on-clear="ModalCtrl.clear('group_uuid')"></bh-clear>
-
-        <!-- @todo make data driven selects components to handle loading and error states -->
-        <ui-select
-          name="group_uuid"
-          ng-model="ModalCtrl.params.group_uuid">
-          <ui-select-match placeholder="{{ 'FORM.SELECT.INVENTORY_GROUP' | translate }}"><span>{{$select.selected.name}}</span></ui-select-match>
-          <ui-select-choices ui-select-focus-patch repeat="group.uuid as group in (ModalCtrl.inventoryGroups | filter:$select.search | orderBy:'name') track by group.uuid">
-            <span ng-bind-html="group.name | highlight:$select.search"></span>
-          </ui-select-choices>
-        </ui-select>
-
-      </div>
-
-      <div class="form-group" ng-class="{ 'has-error' : ModalForm.$submitted && ModalForm.text.$invalid }">
-        <label class="control-label" translate>
-          FORM.LABELS.LABEL
-        </label>
-        <bh-clear on-clear="ModalCtrl.clear('text')"></bh-clear>
-
-        <input type="text" class="form-control" name="text" ng-model="ModalCtrl.params.text">
-        <div class="help-block" ng-messages="ModalForm.text.$error" ng-show="ModalForm.$submitted">
-          <div ng-messages-include="modules/templates/messages.tmpl.html"></div>
+  <div class="modal-body search-modal">
+    <uib-tabset>
+      <uib-tab index="0" heading="{{ 'FORM.LABELS.SEARCH_QUERRIES' | translate }}" data-custom-filter-tab>
+        <div class="tab-body">
+
+          <div class="form-group">
+            <label class="control-label" translate>
+              FORM.LABELS.GROUP
+            </label>
+            <bh-clear on-clear="ModalCtrl.clear('group_uuid')"></bh-clear>
+
+            <!-- @todo make data driven selects components to handle loading and error states -->
+            <ui-select
```

Comment:
We should create a component to manage inventory group separately, so that the controller will not care about inventory group data downloading for focusing on scope model only.
May be we have it already.
If not possible, we can create a separate issue.